tsl/test/expected/backup.out

```diff
@@ -0,0 +1,124 @@
+-------------+-----------+-------+-------------+--------------+------------------+-------------------
+ data_node_1 | localhost | 55432 | db_backup_1 | t            | t                | t
+(1 row)
+
+SELECT * FROM add_data_node('data_node_2', host => 'localhost', database => :'MY_DB2');
+  node_name  |   host    | port  |  database   | node_created | database_created | extension_created 
+-------------+-----------+-------+-------------+--------------+------------------+-------------------
+ data_node_2 | localhost | 55432 | db_backup_2 | t            | t                | t
+(1 row)
+
+SELECT * FROM add_data_node('data_node_3', host => 'localhost', database => :'MY_DB3');
+  node_name  |   host    | port  |  database   | node_created | database_created | extension_created 
+-------------+-----------+-------+-------------+--------------+------------------+-------------------
+ data_node_3 | localhost | 55432 | db_backup_3 | t            | t                | t
+(1 row)
+
+GRANT USAGE ON FOREIGN SERVER data_node_1, data_node_2, data_node_3 TO PUBLIC;
+-- Import testsupport.sql file to data nodes
+\unset ECHO
+-- test NULL (STRICT function)
+SELECT create_distributed_restore_point(NULL);
+ create_distributed_restore_point 
+----------------------------------
+(0 rows)
+
+-- test long restore point name (>= 64 chars)
+\set ON_ERROR_STOP 0
+SELECT create_distributed_restore_point('0123456789012345678901234567890123456789012345678901234567890123456789');
+ERROR:  restore point name is too long
+\set ON_ERROR_STOP 1
+-- test super user check
+\set ON_ERROR_STOP 0
+SET ROLE :ROLE_1;
+SELECT create_distributed_restore_point('test');
+ERROR:  must be superuser to create restore point
+RESET ROLE;
+\set ON_ERROR_STOP 1
+-- make sure 2pc are enabled
+SET timescaledb.enable_2pc = false;
+\set ON_ERROR_STOP 0
+SELECT create_distributed_restore_point('test');
+ERROR:  two-phase commit transactions are not enabled
+\set ON_ERROR_STOP 1
+-- make sure wal_level is replica or logical
+SHOW wal_level;
+ wal_level 
+-----------
+ replica
+(1 row)
+
+-- make sure not in recovery mode
+SELECT pg_is_in_recovery();
+ pg_is_in_recovery 
+-------------------
+ f
+(1 row)
+
+-- test on single node
+CREATE DATABASE dist_rp_test;
+\c dist_rp_test :ROLE_CLUSTER_SUPERUSER
+SET client_min_messages TO ERROR;
+CREATE EXTENSION timescaledb;
+\set ON_ERROR_STOP 0
+SELECT create_distributed_restore_point('test');
+ERROR:  distributed restore point must be created on the access node
+\set ON_ERROR_STOP 1
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+DROP DATABASE dist_rp_test;
+-- test on data node
+\c :MY_DB1 :ROLE_CLUSTER_SUPERUSER;
+\set ON_ERROR_STOP 0
+SELECT create_distributed_restore_point('test');
+ERROR:  distributed restore point must be created on the access node
+\set ON_ERROR_STOP 1
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+-- test on access node
+SELECT node_name, pg_lsn(restore_point) > pg_lsn('0/0') FROM create_distributed_restore_point('dist_rp');
```

Comment:
Should probably add a column alias, although I am not sure what a suitable name for the condition is (I realize it is for making tests reproducible):

```suggestion
SELECT node_name, pg_lsn(restore_point) > pg_lsn('0/0') AS valid_lsn FROM create_distributed_restore_point('dist_rp');
```